Native Facebook Login QuickStarts (#8848)
auth0/docs: articles/quickstart/native/_includes/_facebook_login_prerequisites.md

This tutorial describes how to implement login with the [Facebook SDK](https://developers.facebook.com/docs/).
​
## Before You Start
​
- Install and configure the [Facebook Login SDK](https://developers.facebook.com/docs/facebook-login/). You’ll also go through the process of creating a Facebook app in [https://developers.facebook.com](https://developers.facebook.com). **When you finish this step, you should have an mobile app running with Facebook Login integrated.**  

> - Configure your Auth0 application in the dashboard to use Facebook Native Sign In. See [Add Facebook Login to Native Apps](https://auth0.com/docs/connections/nativesocial/facebook). **When you finish this step, Facebook Native connection should be enabled.**

Reviewer: ```suggestion
- Configure your Auth0 application in the dashboard to use Facebook Native Sign In. See [Add Facebook Login to Native Apps](https://auth0.com/docs/connections/nativesocial/facebook). **When you finish this step, your application will be able to implement Facebook Native Login.**
```
There's no such a thing as a 'Facebook Native' connection
